builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -1063,3 +1066,68 @@ def loadBuildModules(buildModuleUrl){
     }
   }
 }
+
+def servelessDeploy(env, envBucketKey){
+  try {
+    sh "serverless deploy --stage ${env} -v --bucket ${configLoader.AWS.S3[envBucketKey]} --force > output.log"
+    return "success"
+  } catch (ex) {
+    echo "Serverless deployment failed due to $ex"
+  }
+
+  def outputLog = readFile('output.log').trim()
+  return outputLog
+}
+
+
+def isExistingLogGroup (groupName, credsId){
+	def isExistingLogGrp = false
+	def filterLogs = sh (
+		script: "aws logs describe-log-groups --log-group-name-prefix $groupName --output json --profile ${credsId}",
+		returnStdout: true
+	).trim()
+	def toJSON = new groovy.json.JsonSlurper()
+	def filterLogsJSON = toJSON.parseText(filterLogs)
+	def logGroups = filterLogsJSON.logGroups
+	if(logGroups.size() > 0  && logGroups[0].logGroupName == groupName){
+		isExistingLogGrp = true
+	}
+	return isExistingLogGrp
+}
+
+def removeLogGroup(groupName, credsId) {
+	sh "aws logs delete-log-group --log-group-name $groupName --profile ${credsId}"
+	echo "successfully removed log group $groupName"
+}
+
+def handleDeploymentErrors(deployOutput, serviceName, env, credsId) {
```

Comment:
I could see, you are passing envBucketKey and the function defenition contains serviceName. Please correct it.

Reply:
did the changes